README.md

```diff
@@ -417,16 +419,16 @@ This section describes the data format output by the profiler. Some of it is rel
 
 ### Collapsed files
 
-The collapsed file (`.col`) is a [collapsed/folded stacks file](https://github.com/brendangregg/FlameGraph#2-fold-stacks) that'll be written locally per profiling session if gProfiler was invoked with the `-o` switch.  
```

Comment:
Same note as https://github.com/Granulate/gprofiler/pull/680/files#r1103732056 for all these here